alacritty_terminal/src/term/mod.rs

```diff
@@ -1356,7 +1372,12 @@ impl Term {
             }
         }
 
-        urls
+        self.urls = urls;
```

Comment:
Instead of creating a new `Vec` and assigning it here, we can just call `self.urls.clear()` and then push directly to the `Vec` on `Term`.

Reply:
Oh, that's a good idea, thanks!